java/org/contikios/cooja/plugins/TimeLine.java

```diff
@@ -589,11 +591,21 @@ private static double zoomLevelToDivisor(int zoomLevel) {
     return ZOOM_LEVELS[zoomLevel];
   }
 
-  private void zoomFinish (final double zoomDivisor,
+  private void zoomFinish (double zoomDivisor,
                            final long focusTime,
-                           final double focusCenter) {
-    currentPixelDivisor = zoomDivisor;
+                           final double focusCenter) 
+  {
     String note = "";
+  
+    long now = simulation.getSimulationTime();
+    if (now/zoomDivisor > Integer.MAX_VALUE) {
+        // limit zoom to fits sim-time in timeline
+        int min_level = zoomGetLevel( now/Integer.MAX_VALUE );
```

Comment:
Other variables use `camelCase` for naming, and remove the space before/after the method argument.